element-android/pipeline.yml

```diff
@@ -70,7 +70,7 @@ steps:
 
     commands:
       - "mv vector-gplay-universal-*.apk app.apk"
-      - "docker run -v $(pwd)/app.apk:/app.apk --rm -i exodusprivacy/exodus-standalone -j > exodus.json"
+      - "docker run -v $(pwd)/app.apk:/app.apk --rm -i exodusprivacy/exodus-standalone:latest -j /apk.app > exodus.json"
```

Comment:
headdesk; typo in what i wrote after testing locally: "app.apk" not "apk.app" *sigh*